packages/helloworld/package.json

```diff
@@ -23,7 +23,6 @@
     "@react-native/core-cli-utils": "0.77.0-main",
     "@react-native/eslint-config": "0.77.0-main",
     "@react-native/metro-config": "0.77.0-main",
-    "babel-jest": "^29.6.3",
```

Comment:
I think effectively we need babel-jest both here and in devDependencies of `react-native`